docs/project/2024/Mihai-Antonio GEORGESCU/mihai.georgescu2508.md

```diff
@@ -0,0 +1,79 @@
+|2 x [9V Alkaline Battery Varta 6LR61 4922](https://www.optimusdigital.ro/en/9-v-pp3-batteries/3151-9v-alkaline-battery-varta-6lr61-4922-4008496559862.html)|The juice supplying the car |[13 RON](https://www.optimusdigital.ro/en/9-v-pp3-batteries/3151-9v-alkaline-battery-varta-6lr61-4922-4008496559862.html)|
+|1 x [Mounting Bracket for HC-SR04 Ultrasonic Sensor (Red)](https://www.optimusdigital.ro/en/holders-and-mounting-accessories/7104-mounting-bracked-for-hc-sr04-ultrasonic-sensor-red.html?search_query=0104210000045369%09&results=1)|Holds the ultrasonic sensor| [2 RON](https://www.optimusdigital.ro/en/holders-and-mounting-accessories/7104-mounting-bracked-for-hc-sr04-ultrasonic-sensor-red.html?search_query=0104210000045369%09&results=1)|
+|1 x [HC-SR04 Ultrasonic Sensor](https://www.optimusdigital.ro/en/ultrasonic-sensors/9-hc-sr04-ultrasonic-sensor.html?search_query=ultrasonic&results=87)|Radar for the car |[7 RON](https://www.optimusdigital.ro/en/ultrasonic-sensors/9-hc-sr04-ultrasonic-sensor.html?search_query=ultrasonic&results=87)|
+|2 x [400p HQ Breadboard](https://www.optimusdigital.ro/en/breadboards/44-400p-hq-breadboard.html?search_query=breadboard&results=415)|Accomodate hardware devices |[10 RON](https://www.optimusdigital.ro/en/breadboards/44-400p-hq-breadboard.html?search_query=breadboard&results=415)|
+|1 x [Male-Male Wires](https://www.optimusdigital.ro/en/wires-with-connectors/884-set-fire-tata-tata-40p-10-cm.html)|Do the connection |[7 RON](https://www.optimusdigital.ro/en/wires-with-connectors/884-set-fire-tata-tata-40p-10-cm.html)|
+|1 x [FS90 Micro Servomotor with Plastic Reducer](https://www.optimusdigital.ro/en/servomotors/3165-fs90-micro-servomotor-with-plastic-reducing.html?search_query=servomotor&results=111)|Rotates the ultrasonic sensor |[27 RON](https://www.optimusdigital.ro/en/servomotors/3165-fs90-micro-servomotor-with-plastic-reducing.html?search_query=servomotor&results=111)|
+|1 x [5 V Passive Buzzer](https://www.optimusdigital.ro/en/buzzers/634-5v-passive-buzzer.html?search_query=buzzer&results=87)|Honk |[2 RON](https://www.optimusdigital.ro/en/buzzers/634-5v-passive-buzzer.html?search_query=buzzer&results=87)|
+|1 x [10 mm Hexagonal Head Screw](https://www.optimusdigital.ro/en/screws-and-nuts/5756-m6x10-mm-hexagonal-head-screw.html?search_query=screw&results=746)|Fixing the components in place |[1 RON](https://www.optimusdigital.ro/en/screws-and-nuts/5756-m6x10-mm-hexagonal-head-screw.html?search_query=screw&results=746)|
+
+
+## Software
+
+| Library | Description | Usage |
+|---------|-------------|-------|
+| [st7789](https://github.com/almindor/st7789) | Display driver for ST7789 | Used for the display for the Pico Explorer Base |
+| [embedded-graphics](https://github.com/embedded-graphics/embedded-graphics) | 2D graphics library | Used for drawing to the display |
```

Comment:
Do you need these two libraries?

Reply:
no, i forgot to delete them from the template....... sorry🙃🙃🙃🙃🙃